anti-analysis/anti-debugging/debugger-detection/check-for-software-breakpoints.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: check for software breakpoints
+    namespace: anti-analysis/anti-debugging/debugger-detection
+    author: michael.hunhoff@fireeye.com
+    scope: function
+    mbc:
+      - Anti-Behavioral Analysis::Detect Debugger
+    references:
+      - https://github.com/LordNoteworthy/al-khaser/blob/master/al-khaser/AntiDebug/SoftwareBreakpoints.cpp
+    examples:
+      - al-khaser_x86.exe_:0x431020
+  features:
+    - and:
+      - basic block:
+        - and:
+          - mnemonic: cmp
+          - number: 0xCC
```

Comment:
could add the less-used opcode `CD 03` 

`CD       ib  INT imm8                Interrupt vector number (imm8)`